pallets/asset/src/tests.rs

```diff
@@ -0,0 +1,84 @@
+use super::*;
```

Comment:
does this need `#[cfg(test)]` in lib.rs?

Reply:
Yes it is required. Have added in lib.rs .